x/consensus/types/keys.go

```diff
@@ -2,7 +2,7 @@ package types
 
 const (
 	// ModuleName defines the module name
-	ModuleName = "consensus"
```

Comment:
does the upgrade handler for this chain handle this change? its a bit of a scary thing

Reply:
Yes. https://github.com/palomachain/paloma/pull/797/files#diff-aeec7ddb4cf8b7e9e6e57625770d369cce5f377cc350b42c3aca0427bba8dfbcR160